code/game/jobs/job/security.dm

```diff
@@ -22,6 +22,7 @@
 	exp_type = EXP_TYPE_SECURITY
 	disabilities_allowed = 0
 	outfit = /datum/outfit/job/hos
+	important_information = "This role requires you to coordinate a large department. You are required to be familiar with Standard Operating Procedure (Department), basic job duties, and must act in a professional and reasonable manner (roleplay)."
```

Comment:
Looking at this compared to other ones in the file, I feel like these should be shorter and to the point.

Reply:
"This role requires you to coordinate a department. You are required to be familiar with Standard Operating Procedure (Department), basic job duties, and act professionally (roleplay)."